source/blog/2017-12-08-ovirt-ansible-roles-split.html.md

```diff
@@ -0,0 +1,104 @@
+
+![oVirt roles in Ansible Galaxy](galaxy_roles.png)
+
+## How to install the roles 
+There are still two posibilities of installing the roles, you can do it either via Ansible Galaxy or from oVirt repository.
+
+### Ansible Galaxy
+Newly you can now install just a single role, instead of downloading them all at once.
+For example to install just oVirt cluster upgrade role, you have to run following command:
+
+```bash
+$ ansible-galaxy install oVirt.cluster-upgrade
+```
+
+Note that we still support the posibility to download and install all roles at once,
+you can do it executing the following command:
+
+```bash
+$ ansible-galaxy install oVirt.ovirt-ansible-roles
+```
+
+### From oVirt repository
+Also via RPM you can now install just a single role, instead of downloading them all at once.
+For example to install just oVirt cluster upgrade role, you have to run following command:
+
+```bash
+$ yum install ovirt-ansible-cluster-upgrade
+```
+
+Note that we didn't obsoleted the `ovirt-ansible-roles` package, this package still exists
+and this package still install all available roles:
+
+```bash
+$ yum install ovirt-ansible-roles
+```
+
+The only difference is that this package now requires all the separate packages, so the packages itself
+only provides documentation and examples.
+
+## Compatibility changes
+Unfortunatelly the split has its price.
+
+### The names
+
+#### Roles from Ansible Galaxy
+Unfortunatelly split has also broke the names of the roles in Ansible Galaxy.
+So before user could used the following names:
+
+ - ovirt-cluster-upgrade
+ - ovirt-image-template
+ - ovirt-infra
+ - ovirt-manageiq
+ - ovirt-vm-infra
+
+But since the naming in Ansible Galaxy is done as {owner}.{role-name}, we had to rename
+the roles to:
+
+ - oVirt.cluster-upgrade
+ - oVirt.image-template
+ - oVirt.infra
+ - oVirt.manageiq
+ - oVirt.vm-infra
+
+Notice that we have changed the first dash by the dot and renamed `ovirt` to `oVirt`.
+
+#### Roles from RPM
+For the users who installed roles from RPM. The names with dash will still work,
+as we are creating a symlink to new names. But please be aware that we will remove
+this legacy name one day in future.
+
+
+### Removal of component roles
+Previously the ovirt-ansible-roles package shipped following compoenent roles:
+
+ - ovirt-aaa-jdbc
+ - ovirt-affinity-groups
+ - ovirt-clusters
+ - ovirt-datacenter-cleanup
+ - ovirt-datacenters
+ - ovirt-external-providers
+ - ovirt-hosts
+ - ovirt-mac-pools
+ - ovirt-networks
+ - ovirt-permissions
+ - ovirt-storages
+
+Those roles are not provided as sepratelly installable roles, but those are just internal
+roles of `ovirt-infra` and `ovirt-vm-infra` roles. Please note that any of those role can
```

Comment:
Isn't it worth to add an example how to run them as a task?

Reply:
Ok, will add.